suite-native/accounts/src/selectors.ts

```diff
@@ -32,18 +35,34 @@ export const isFilterValueMatchingAccount = (account: Account, filterValue: stri
 /**
  * Filter accounts by labels, network names and included token names.
  */
-const filterAccountsByLabelAndNetworkNames = (accounts: Account[], filterValue?: string | null) => {
+const filterAccountsByLabelAndNetworkNames = (
+    accounts: readonly Account[],
```

Comment:
Is the readonly really necessary? And why are we using it in some selectors and not others? I'd drop it as it's not really useful and you get a runtime error when trying to modify readonly state anyway

Reply:
I am afraid it has to be `readonly`. This is (IMO) very annoying feature of `ts-belt` that everything that comes out of any of its functions is always set with `readonly` flag. If I set the `accounts` param without readonly, the TSlint does not like it.